spec/lib/gems/baw_workers/upload_service_steps.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
```

Comment:
Some of these tests have no expectations, are they just testing if the function will crash or not?

Reply:
Yeah, some are just setup steps. Some do tests.